apps/cookbook/src/app/examples/examples.common.ts

```diff
@@ -1,65 +1,65 @@
 import { ICON_SETTINGS, IconSettings } from '@kirbydesign/designsystem';
```

Comment:
I am unsure why changed lines in examples.common.ts is here. It was committed when mergíng master into this branch. Should this be reverted?